src/ray/core_worker/task_execution.cc

```diff
@@ -29,7 +29,7 @@ Status CoreWorkerTaskExecutionInterface::Run(const TaskExecutor &executor) {
       const auto &spec = task.GetTaskSpecification();
       core_worker_.worker_context_.SetCurrentTask(spec);
 
-      ray::rpc::Language language = ToTaskLanguage(spec.GetLanguage());
+      ray::rpc::Language language = ToRpcTaskLanguage(spec.GetLanguage());
```

Comment:
nit: `RPC` not `Rpc`
Unless there's a convention of doing pure camel case even for initialisms in Ray - I'm not sure!

Reply:
I think both naming conventions are being used widely. I googled this a while ago, a more usual practice is to use all upper cases if the the acronym is less than 2 letters (e.g., `IO`), and use camel case otherwise (e.g., `Rpc`).
Note, currently we use `TaskID` in c++ and Python, and `TaskId` in Java. That is because all Java linters prefer camel case regardless of word length.

Reply:
I just took a look. Google's C++ style guide suggest `Rpc`.

> For some symbols, this style guide recommends names to start with a capital letter and to have a capital letter for each new word (a.k.a. "Camel Case" or "Pascal case"). When abbreviations or acronyms appear in such names, prefer to capitalize the abbreviations or acronyms as single words (i.e StartRpc(), not StartRPC()).

https://google.github.io/styleguide/cppguide.html#Naming